src/lexer/Lexer.spec.ts

```diff
@@ -1222,4 +1222,40 @@ describe('lexer', () => {
             TokenKind.Eof
         ]);
     });
+
+    describe('regular expression literals', () => {
+        function testRegex(...regexps: Array<string | RegExp>) {
+            regexps = regexps.map(x => x.toString());
+            const results = [] as string[];
+            for (const regexp of regexps) {
+                const { tokens } = Lexer.scan(regexp as string);
+                results.push(tokens[0].text);
+            }
+            expect(results).to.eql(regexps);
+        }
+
+        it('recognizes regex literals', () => {
+            testRegex(
+                /simple/,
+                /SimpleWithValidFlags/g,
+                /UnknownFlags/gi,
```

Comment:
One case should probably have a double quote to verify it's correctly escaped

Reply:
OH! I knew I was forgetting something. I just pushed a commit  that verifies the lexer handles this fine. However, the parser does _not_ properly escape the quotemark. Nice catch! I'll work on that now.

Reply:
Fixed.